src/cli_plugin/install/kibana.js

```diff
@@ -19,34 +17,28 @@ export function existingInstall(settings, logger) {
 
 export async function rebuildCache(settings, logger) {
   logger.log('Optimizing and caching browser bundles...');
-  const serverConfig = _.merge(
-    readYamlConfig(settings.config),
-    {
-      env: 'production',
-      logging: {
-        silent: settings.silent,
-        quiet: !settings.silent,
-        verbose: false
-      },
-      optimize: {
-        useBundleCache: false
-      },
-      server: {
-        autoListen: false
-      },
-      plugins: {
-        initialize: false,
-        scanDirs: [settings.pluginDir, fromRoot('src/core_plugins')]
-      },
-      uiSettings: {
-        enabled: false
-      }
-    }
-  );
-
-  const kbnServer = new KbnServer(serverConfig);
-  await kbnServer.ready();
-  await kbnServer.close();
+
+  const kibanaScript = /^win/.test(process.platform)
+    ? '.\\bin\\kibana.bat'
+    : './bin/kibana';
+
+  const kibanaArgs = [
+    '--env.name=production',
+    `--logging.silent=${settings.silent}`,
+    `--logging.quiet=${!settings.silent}`,
+    `--logging.verbose=false`,
+    '--optimize.useBundleCache=false',
+    '--server.autoListen=false',
+    '--plugins.initialize=false',
+    '--uiSettings.enabled=false'
+  ];
+
+  const proc = execa(kibanaScript, kibanaArgs, {
+    stdio: ['ignore', 'pipe', 'pipe'],
```

Comment:
nit, if we're piping thoughts on catching the await?
```
Plugin installation was unsuccessful due to error "Command failed: ./bin/kibana --env.name=production --logging.silent=false --logging.quiet=true --logging.verbose=false --optimize.useBundleCache=false --server.autoListen=false --plugins.initialize=false --uiSettings.enabled=false
```

Reply:
hmm, is there actually an error?

Reply:
Assuming you are trying to avoid duplicate error logging, I think this pairs with the fact that Optimization is never actually completing. Instead of logging I think we should disable logging and rely on the execa error message to show the user stdout.


When I manually disabled the logging using:

```
await watchStdioForLine(proc, { log(){} }, 'log', /Optimization .+ complete/);
```

And forced an error, this is the output I got: 

![image](https://user-images.githubusercontent.com/1329312/40551394-a404daa0-5ff1-11e8-91b1-265f17ec8091.png)